packages/monaco/src/browser/monaco-workspace.ts

```diff
@@ -262,17 +269,17 @@ export class MonacoWorkspace implements lang.Workspace {
     }
 
     async applyBulkEdit(workspaceEdit: monaco.languages.WorkspaceEdit, options?: EditorOpenerOptions): monaco.Promise<monaco.editor.IBulkEditResult> {
-        let totalEdits = 0;
-        let totalFiles = 0;
-        const uri2Edits = this.groupEdits(workspaceEdit);
-        for (const uri of uri2Edits.keys()) {
-            const editorWidget = await this.editorManager.open(new URI(uri), options);
-            const editor = MonacoEditor.get(editorWidget);
-            if (editor) {
-                const model = editor.document.textEditorModel;
-                const currentSelections = editor.getControl().getSelections();
-                const edits = uri2Edits.get(uri)!;
-                const editOperations: monaco.editor.IIdentifiedSingleEditOperation[] = edits.map(edit => ({
+        try {
+            const unresolvedEditorEdits = this.groupEdits(workspaceEdit);
+            const editorEdits = await this.openEditors(unresolvedEditorEdits, options);
```

Comment:
Somehow this line never returns, what I did:

- Open `theia/packages/core/src/browser/common-frontend-contribution.ts`
- Right click on `Command` symbol (line 69) and do `Rename Symbol`
- Rename to `Command1`

In my case, all the editors still open in sequence, but worse than that it doesn't actually change anything.

After placing a few breakpoints, it seems like this line (274) is never returning from the `await`.

Reply:
Simplified it and now it works.